Duplicati/Library/Backend/GoogleServices/GoogleCloudStorage.cs

```diff
@@ -249,37 +259,46 @@ public void Get(string remotename, System.IO.Stream stream)
         {
             try
             {
-                var url = WebApi.GoogleCloudStorage.GetUrl(m_bucket, Library.Utility.Uri.UrlPathEncode(m_prefix + remotename)); 
-                var req = m_oauth.CreateRequest(url);
-                var areq = new AsyncHttpRequest(req);
+                string url = WebApi.GoogleCloudStorage.GetUrl(m_bucket, Library.Utility.Uri.UrlPathEncode(m_prefix + remotename));
+                HttpWebRequest req = m_oauth.CreateRequest(url);
+                AsyncHttpRequest areq = new AsyncHttpRequest(req);
 
-                using (var resp = areq.GetResponse())
-                using (var rs = areq.GetResponseStream())
+                using (WebResponse resp = areq.GetResponse())
+                using (System.IO.Stream rs = areq.GetResponseStream())
                     Library.Utility.Utility.CopyStream(rs, stream);
             }
             catch (WebException wex)
             {
                 if (wex.Response is HttpWebResponse response && response.StatusCode == HttpStatusCode.NotFound)
                     throw new FileMissingException();
                 else
-                    throw;
+                {
+                    using (Stream exstream = wex.Response.GetResponseStream())

```

Comment:
I don't think you can guarantee there to be a response stream present? If the error is "Network disconnected" there will be no response stream and the user will get an exception here.